modin/pandas/series.py

```diff
@@ -2240,3 +2240,76 @@ def _default_to_pandas(self, op, *args, **kwargs):
         return self._series._default_to_pandas(
             lambda series: op(series.str, *args, **kwargs)
         )
+
+
+class CategoryMethods(object):
+    def __init__(self, series):
+        self._series = series
+        self._query_compiler = series._query_compiler
+
+    @property
+    def categories(self):
+        return self._series._default_to_pandas(pandas.Series.cat).categories
```

Comment:
Why didn't you add tests for this?

Reply:
I assumed `df_equals` would implicitly check that since it compares column's categories.

Reply:
I think it's better to have explicit tests checking a functionality instead of to rely on implicit calls. Lets add a test for this.

Reply:
Alright, thanks!